server/js/getManyFromDB.js

```diff
@@ -0,0 +1,11 @@
+import mongoose from "mongoose";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+export default async function getManyFromDB(url, collection, sortBy, limit) {
+    const conn = await mongoose.connect(url);
+    const highScores = await collection.find().lean().sort(sortBy).limit(limit);
+    await conn.disconnect();
```

Comment:
You should generally never need to disconnect from mongo. Keeping the connections open means they can be reused which will be faster.